src/write_eval_config.R

```diff
@@ -72,8 +72,17 @@ write_eval_config <- function(locations, forecast_dates,
   retro_rt_path <- file.path("input", "retro_Rt", "Rt_draws.parquet")
   score_subdir <- file.path("output", "eval", "hub")
   hub_model_names <- c(
```

Comment:
If the goal is to use all of the models submitted to the hub for the period of interest (and then filter for completeness), ideally we should query the hub itself for that information rather than hard coding it.

Reply:
Do we have a better way of querying than loading in the data from github? I talked to @ekr-cfa ad @k88hudson and they had said that they just pull from github. I thought the hubverse stuff was still in development, Happy to be proven wrong. But otherwise, I don't think we want to pull in all of it because theres a ton and only certain teams have submitted recently